src/moj/components/cookie-banner/_cookie-banner.scss

```diff
@@ -0,0 +1,36 @@
+.moj-cookie-banner {
+  display: none;
+  @include govuk-font(16);
+
+  box-sizing: border-box;
+
+  padding-top: govuk-spacing(3);
+  padding-bottom: govuk-spacing(3);
+  left: govuk-spacing(3);
+  padding-right: govuk-spacing(3);
+  background-color: lighten(
+    desaturate(govuk-colour("light-blue"), 8.46),
+    42.55
```

Comment:
I'm not a fan of these magic numbers inside the component. Think we should either use an existing govuk colour, or [add a new one to the MOJDS](https://github.com/ministryofjustice/moj-frontend/blob/master/src/moj/settings/_colours.scss) so that it's formally defined.

Reply:
Good call, have changed this to the GDS light-grey standard